srcpkgs/hunspell-sv_SE/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'hunspell-sv_SE'
+pkgname=hunspell-sv_SE
+version=2.41
+revision=1
+noarch=yes
+create_wrksrc=yes
+hostmakedepends="unzip"
+short_desc="Swedish sv_SE dictionary for hunspell"
+maintainer="Venca Uruba <uruba@outlook.com>"
+license="GNU-GPL-v3, GNU-GPL-v2, LGPL-v3+"
```

Comment:
Please use the spdx short-identifier for the licenses.

LGPL-v3+ is LGPL-3.0-or-later

Please check whether GNU-GPL-v3 correspond to GPL-3.0-or-later or GPL-3.0-only
Please check whether GNU-GPL-v2 correspond to GPL-2.0-or-later or GPL-2.0-only